core/routes/oauth_schema.json

```diff
@@ -11,7 +11,8 @@
                 "AUTH_FACEBOOK_CLIENT_SECRET": {
                     "type": "string",
```

Comment:
Lets pull all schema jsons in different folder. Instead of having them in routes